### DISCUSSION TITLE
When em.upsert() wil be released?

### DISCUSSION THREAD

**Discussion by Nisgrak** [mikro-orm/mikro-orm#3627] [General]
Hi!

A some time ago I see you posted a tweet about `em.upsert()` https://github.com/mikro-orm/mikro-orm/pull/3525 but I think you doesn't ship a new release with that, when will be out?

Thanks!

**Comment by B4nan** [comment]
When the time comes, I really dont know, there are quite a lot of issues I want to deal with first. 

You can use the dev version if you want it, every commit to master is released.

**Comment by Nisgrak** [comment]
I didn't know the dev version was available thanks for all your work ❤️

**Reply by Langstra** [reply]
By the way, as of `5.5.0` it is released. https://github.com/mikro-orm/mikro-orm/releases/tag/v5.5.0
